manager/integration/tests/test_ha.py

```diff
@@ -2365,9 +2354,76 @@ def test_replica_failure_during_attaching():
     9. Attach volume2.
        --> Verify volume will be attached with state Degraded.
     10. Wait for the replenishment interval.
-        --> Verify the failed replica using the extra disk will be removed.
+        ## --> Verify the failed replica using the extra disk will be removed.
```

Comment:
You can directly remove it... 
In general, LGTM

Reply:
fixed